pkg/reconciler/revision/resources/queue_test.go

```diff
@@ -885,6 +885,7 @@ var defaultEnv = map[string]string{
 	"METRICS_DOMAIN":                   metrics.Domain(),
 	"METRICS_COLLECTOR_ADDRESS":        "",
 	"QUEUE_SERVING_PORT":               "8012",
+	"QUEUE_SERVING_TLS_PORT":           "8112",
```

Comment:
Use `networking.BackendHTTPSPort`?

Reply:
`QUEUE_SERVING_PORT` does not use the const so I leave it as it is.